config/sections/files.php

```diff
@@ -195,7 +195,6 @@
                 'link'     => $this->link(),
                 'max'      => $this->max,
                 'min'      => $this->min,
-                'query'    => $this->query,
```

Comment:
@afbora I didn't see the need to pass this back to the frontend - do you see any?

Reply:
This was for default search term/query. But we removed this feature while PR in progress but we forgot this line. So OK.